frontend/Plugin.js

```diff
@@ -0,0 +1,173 @@
+import SgTrackingPlugin from '@shopgate/tracking-core/plugins/Base';
```

Comment:
This module wasn't installed to my `node_modules` - not sure right now, if this is an issue.